examples/grant-access/src/grant-access.ts

```diff
@@ -65,12 +71,10 @@ app.get("/manage", async (req, res) => {
         <p>the requested access ?</p>
       </div>
       <div>
-        <input type="hidden" name="requestVc" id="requestVc" value=${
-          req.query.requestVc
-        }>
-        <input type="hidden" name="redirectUrl" id="redirectUrl" value=${
-          req.query.redirectUrl
-        }>
+        <input type="hidden" name="requestVc" id="requestVc" value=${JSON.stringify(
+          accessRequest
+        )}>
```

Comment:
Oh yeah I mean the way HTML is templated here is not done according to best practices at all for the sake of simplicity/compacity. I'm happy to change that to something better if you think this could set a bad precedent.

Reply:
Perhaps for now we just add a warning in the code (e.g., a comment) saying "you probably shouldn't do this", or just replace `JSON.stringify` with `serialize-javascript`, but that does mean pulling in another dependency